sdk/metricsadvisor/azure-ai-metricsadvisor/README.md

```diff
@@ -48,6 +48,25 @@ admin_client = MetricsAdvisorAdministrationClient(service_endpoint,
 
 ## Key concepts
 
+### MetricsAdvisorClient
+
+`MetricsAdvisorClient` helps with:
+
+- listing incidents
+- listing root causes of incidents
+- retrieving original time series data and time series data enriched by the service.
+- listing alerts
+- adding feedback to tune your model
+
+### MetricsAdvisorAdministrationClient
+
+`MetricsAdvisorAdministrationClient` allows you to
+
+- managing data feeds
+- configuring anomaly detection configurations
+- configuring anomaly alerting configurations
+- managing hooks
```

Comment:
The "allows you to" part makes it read better if we use "manage" and "configure" for the bullet points